stage1_fly/aci/aci-manifest.in

```diff
@@ -32,6 +32,10 @@
         {
             "name": "coreos.com/rkt/stage1/stop",
             "value": "/stop"
+        },
+        {
+            "name": "coreos.com/rkt/stage1/interface-version",
+            "value": "4"
```

Comment:
I really need to bump the version - @iaguis would it be OK if I accept `--hostname` but ignore it? Should I warn?